### PR TITLE
Blog post about Kafka and Cloud Events

### PR DESCRIPTION
**Pull request by cescoffier**
A blog post about Kafka and Cloud Events.

- [x] Do not merge until the date is updated.

### CODE REVIEW

**Comment by cescoffier** [comment]
CC @jponge as Cloud Event expert.

File: _posts/2020-12-01-kafka-cloud-events.adoc [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+First, let's look at the why.
+Events are everywhere.
+Many modern systems are using events one way or another.
+Events can be used to implement event sourcing, communicate facts, trigger out-of-band processing, or send notifications.
+Events become an essential piece of any system.
+
+Yet event publishers tend to describe events differently.
+I don't mean the content is different, but the envelope and the event's format are heterogeneous, even if these events transit on the same event mesh.
+Some applications choose JSON and encode everything in the event's payload; some other systems prefer binary formats, such as Avro or Protobuf, and use the protocol capability, such as headers or properties,  to transport metadata about the wrapped payload.
```
Comment (kenfinnigan) [inline _posts/2020-12-01-kafka-cloud-events.adoc:26]: Extra space after "headers or properties,"

File: _posts/2020-12-01-kafka-cloud-events.adoc [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+While event-driven architecture claims to ease the integration with external systems, this disparity is doing the opposite.
+It's not rare to need a specific event translator with the only purpose to adapt events from one format to another.
+
+So, what's Cloud Event?
+Cloud Event proposes a common way to describe events.
+The goal is, obviously, interoperability and facilitating the integration burden.
```
Comment (kenfinnigan) [inline _posts/2020-12-01-kafka-cloud-events.adoc:32]: Maybe "facilitating" should be "easing", as you're saying the integration burden should be easier and not harder

File: _posts/2020-12-01-kafka-cloud-events.adoc [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+To write your outgoing messages as Cloud Event, you only need to specify the `cloud-events-type` and `cloud-events-source` attributes on your channel:
+
+[source, properties]
+----
+mp.messaging.outgoing.generated-price.connector=smallrye-kafka
+mp.messaging.outgoing.generated-price.topic=prices
+mp.messaging.outgoing.generated-price.value.serializer=org.apache.kafka.common.serialization.IntegerSerializer
+mp.messaging.outgoing.generated-price.cloud-events-source=price-generators
+mp.messaging.outgoing.generated-price.cloud-events-type=price
+mp.messaging.outgoing.generated-price.cloud-events-subject=generated-prices
```
Comment (kenfinnigan) [inline _posts/2020-12-01-kafka-cloud-events.adoc:128]: Do you need to describe what the `subject` key is for?

**Review by kenfinnigan — APPROVED** [review]
Couple of minor comments

File: _posts/2020-12-01-kafka-cloud-events.adoc (hunk 3 above)
Comment (cescoffier) [reply _posts/2020-12-01-kafka-cloud-events.adoc:128]: Oh, good catch! Forgot I added this one in my example.

**Comment by cescoffier** [comment]
@kenfinnigan All done!

**Comment by kenfinnigan** [comment]
Looks good. Pull the trigger when you're ready

**Pull request merged**